Adyen/Model/PaymentComponentData.swift

```diff
@@ -116,8 +121,22 @@ public struct PaymentComponentData {
                              order: order,
                              storePaymentMethod: storePaymentMethod,
                              browserInfo: browserInfo,
+                             checkoutAttemptId: checkoutAttemptId,
                              installments: installments)
     }
+
+    @_spi(AdyenInternal)
+    public func replacingCheckoutAttemptID(with checkoutAttemptId: String?) -> PaymentComponentData {
```

Comment:
'replacingCheckoutAttemptID' is old school :-), can it be:

```
replacing(checkoutAttemptId: String?) -> PaymentComponentData
```
```

Reply:
Will do in another PR 😛